src/graphs.py

```diff
@@ -0,0 +1,222 @@
+from matplotlib import pyplot as plt
+from matplotlib import tri as tri
+from matplotlib import cm
```

Comment:
The standard way to import these is:
```suggestion
import matplotlib.pyplot as plt
import matplotlib.tri as tri
import matplotlib.cm as cm
```